UPGRADE-3.1.md

```diff
@@ -24,6 +24,22 @@ FrameworkBundle
 
  * As it was never an officially supported feature, the support for absolute
    template paths has been deprecated and will be removed in Symfony 4.0.
+ * Since the refactoring of 2.8, form types without dependencies are autoloaded
+   by the `DependencyInjectionExtension` in the form registry.
+   Consequently they are deprecated and will be removed in 4.0.
```

Comment:
> form types without dependencies are autoloaded by the `DependencyInjectionExtension` only, through the form registry

this sentence is wrong. The `DependencyInjectionExtension` has nothing to do in the auto-registration in Symfony 2.8+. This extension is about the loading from services